sia_js/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "sia_js"
+version = "0.1.0"
+authors = ["The Sia Foundation <hi@sia.tech>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = []
+test_in_browser = []
```

Comment:
Do we need this. Doesn't seem very nice to require our lib to expose a feature just for testing. Maybe we can get away with using the `WASM_BINDGEN_USE_BROWSER` environment variable somehow?

Reply:
🤷🏻